src/iOS/Xamarin.iOS/Samples/GeometryEngine/SpatialOperations/README.md

```diff
@@ -1,9 +0,0 @@
-# Spatial operations
```

Comment:
Why is the README getting deleted?

Reply:
It doesn't matter, all readmes will be regenerated in an unpcoming PR. That whole sample was excluded from the project (and I think all projects) due to a merge gone wrong.